examples/wifi-echo/server/esp32/main/gen/call-command-handler.c

```diff
@@ -128,8 +174,10 @@ EmberAfStatus emberAfClusterSpecificCommandParse(EmberAfClusterCommand * cmd)
         }
     }
     else if (cmd->direction == (uint8_t) ZCL_DIRECTION_CLIENT_TO_SERVER &&
-             emberAfContainsServerWithMfgCode(cmd->apsFrame->destinationEndpoint, cmd->apsFrame->clusterId, cmd->mfgCode))
+             (emberAfContainsServerWithMfgCode(cmd->apsFrame->destinationEndpoint, cmd->apsFrame->clusterId, cmd->mfgCode) ||
+              cmd->apsFrame->clusterId == ZCL_ECHO_CLUSTER_ID))
```

Comment:
This is the dirty part that bypass some checks to see if it is part of a endpoint or not...